docs/usage/configuration-options.md

```diff
@@ -702,6 +702,18 @@ It is pointless to edit the labels, as Renovate bot restores the labels on each
 
 Configure this option if you prefer a different title for the Dependency Dashboard.
 
+## dependencyDashboardOSVVulnerabilitySummary
+
+Use this option to control if the Dependency Dashboard should have a section that shows the CVEs for your repository.
```

Comment:
Needs clarity about the dependency/limitation of it being the osv alerts and not github's